gallery/src/styles/NxCard/NxCardHorizontalExample.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faShapes, faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+import { NxFontAwesomeIcon, NxButton } from '@sonatype/react-shared-components';
+
+export default function NxCardHorizontalExample() {
+  return (
+    <div className="nx-card-container nx-card-container--column">
+      <section className="nx-card">
+        <header className="nx-card__header">
+          <h3 className="nx-h3">Header</h3>
+        </header>
+        <div className="nx-card__call-out">
+          <span className="nx-card__call-out-text">42</span>
+        </div>
+        <div className="nx-card__content">
+          Text
+        </div>
+        <footer className="nx-card__footer">
+          <a href="#" className="nx-text-link">Link</a>
+        </footer>
+      </section>
+      <section className="nx-card">
+        <header className="nx-card__header">
+          <h3 className="nx-h3">Card header</h3>
+        </header>
+        <div className="nx-card__call-out">
+          <NxFontAwesomeIcon icon={faShapes} />
+        </div>
+        <div className="nx-card__content">
+          Data point details
+        </div>
+        <footer className="nx-card__footer">
+          <div className="nx-btn-bar">
+            <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faEdit} className="nx-icon--large" /></NxButton>
+          </div>
+        </footer>
+      </section>
+      <section className="nx-card">
+        <header className="nx-card__header">
+          <h3 className="nx-h3">Card header</h3>
+        </header>
+        <div className="nx-card__call-out">
+          <NxFontAwesomeIcon icon={faShapes} className="nx-icon--large" />
```

Comment:
Is it valid BEM, or a good idea, to have a modifier on a different block that is only defined in a specific context like this?  I'd think we'd instead want something tied to this block, like `.nx-card__large-call-out-icon`

Reply:
> Is it valid BEM, or a good idea, to have a modifier on a different block that is only defined in a specific context like this? I'd think we'd instead want something tied to this block, like `.nx-card__large-call-out-icon`

What was in my mind when I did this is whether we would want to standardize these icon sizes in some way (which still makes this wrong since the `48px` icon should also have a size related class.

Reply:
> > Is it valid BEM, or a good idea, to have a modifier on a different block that is only defined in a specific context like this? I'd think we'd instead want something tied to this block, like `.nx-card__large-call-out-icon`
> 
> What was in my mind when I did this is whether we would want to standardize these icon sizes in some way (which still makes this wrong since the `48px` icon should also have a size related class.

In the end I clarified the classes for the icons but left them in rather than putting them inline or in the gallery. I think that there is a reasonable change that icons will be used in the call-out. https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/cd45aea566dbdab783874f540aa041f78f82fdf2